openmc/material.py

```diff
@@ -973,3 +973,73 @@ def export_to_xml(self, path='materials.xml'):
         # Write the XML Tree to the materials.xml file
         tree = ET.ElementTree(root_element)
         tree.write(path, xml_declaration=True, encoding='utf-8')
+
+
+def make_boric_acid(boron_ppm, temperature=293., pressure=0.1013, density=None,
```

Comment:
Also, this function is awesomely useful for just creating water too.  Could it be generalized so that it creates water at a given T & P, with an optional boron ppm parameter?

Reply:
I think we should leave the borated water function separate from a hypothetical water from T & P function.  The explicitly-named function is probably easier for users to find and easier to mentally parse in a block of code.